tools/grab_spotify_saved_tracks.py

```
@@ -111,95 +112,85 @@ def generate_unique_slug(title, date_str, url):
         sp = setup_spotify()
-        
-        # Fetch tracks
         tracks = get_saved_tracks(sp)
-        
-        # Create content files
+
         new_tracks_count = 0
         for track in tracks:
             if create_hugo_content(track, hugo_content_dir):
                 new_tracks_count += 1
                 logging.info(f"Created new track post: {track['title']}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)